google/cloud/logging_v2/handlers/_helpers.py

```diff
@@ -211,3 +235,37 @@ def get_request_data():
             return http_request, trace_id, span_id, trace_sampled
 
     return None, None, None, False
+
+
+def get_request_and_trace_data():
```

Comment:
Does this need to be a new function? The docstrings of `get_request_data` says "Helper to get http_request and trace data from supported web frameworks". It seems to me like this logic should just be merged into the existing one

Reply:
I was concerned that by changing the function name that it could potentially be a breaking change. It should be OK to change the function name because it's in a private module, right?

Reply:
Yes, the module is private, so it should be safe

Does the function need to be renamed though? Why not add new functionality to the existing function?